var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -153,6 +155,9 @@ def get_cmake_args(self):
 
         nmodl_options = 'codegen --force'
 
+        if spec.satisfies('~codegenopt'):
```

Comment:
Don’t we need to pass true flag explicitly? I am thinking now that the codegenopt should be false on nmodl side by default (if it’s not the case by default).

Reply:
I wanted to remain as close as possible to the current state. It was always true before. Now, if we remove it, we do something. A little bit like ~report I think

Reply:
I am thinking of making it false default because with the current setting naive user would get incorrect current traces. Hence making optimization flag optional would make more sense?

Reply:
That is true indeed. In this case I need to change nmodl too